main/src/ui/conversation_view_controller.vala

```diff
@@ -178,10 +178,12 @@ public class ConversationViewController : Object {
     private void update_conversation_topic(string? subtitle = null) {
         if (subtitle != null) {
             conversation_topic = Util.summarize_whitespaces_to_space(subtitle);
+            conversation_topic = Util.parse_add_markup(conversation_topic, null, true, false);
```

Comment:
You could also set `parse_text_markup = true`

Reply:
@fiaxh done setting `parse_text_markup = true`